DOCS.md

```diff
@@ -48,7 +48,7 @@ Similar to Navbars, footers are also managed as snippets and also render top dow
 A django app called `website` has been created to hold your custom changes. In website/static/ there are custom.css and custom.js files that get loaded on every page by default. Adding anything to these files will automatically populate on the site and override any default styles. By default, Bootstrap 4 and jQuery are already included on the site.
 
 ### Custom templates
-The templates directory inside the `website` app is empty by default. Any templates you put in here will override the default coderedcms templates if they follow the same name and diretory structure. This uses the standard Django template rendering engine. For example, to change the formatting of the article page, copy `coderedcms/templates/coderedcms/pages/article_page.html` to `website/templates/coderedcms/pages/article_page.html` and modify it.
+The templates directory inside the `website` app is empty by default. Any templates you put in here will override the default coderedcms templates if they follow the same name and diretory structure. This uses the standard Django template rendering engine. For example, to change the formatting of the article page, copy `core/templates/core/pages/article_page.html` to `website/templates/core/pages/article_page.html` and modify it.
```

Comment:
General comment that applies across the whole pull request. I do not like that "coderedcms" is no longer part of the template path. "core" is such a generic word that it could easily conflict with other pip packages. Also, from the client project perspective, it is not very clear that templates in "core" are part of coderedcms.

I do like the current naming convention, because when looking at a project that uses coderedcms, it is readily apparent that templates in the coderedcms directory are overriding the defaults. If anything, I would propose keeping it as-is or updating to "coderedcms/core/"